src/server.e2e.ts

```diff
@@ -0,0 +1,17 @@
+import { bootstrap } from '@wix/yoshi-serverless-testing';
+import HttpClient from '@wix/yoshi-serverless-client';
+import { fetchComments } from './api/comments.api';
+
+const serverlessApp = bootstrap();
+serverlessApp.beforeAndAfter();
+let client: HttpClient;
+
+beforeAll(async () => {
+  client = new HttpClient({ baseUrl: serverlessApp.getUrl() });
+});
+
+test('should fetch comments', async () => {
+  const response = await client.request(fetchComments)();
+  console.log('Test Comments Response:', response);
+  expect(response.greet).toBe([]);
```

Comment:
I assume you've chose to expect empty array because you were unable to fetch comments? 
I recommend checking out the [ambassador testkit documentation](https://github.com/wix-private/ambassador-testkit). It has an example for a similar use case- fetching comments.